pyproject.toml

```diff
@@ -20,6 +20,11 @@ flask-sqlalchemy = "^3.1.1"
 faker = "^25.0.0"
 flask-login = "^0.6.3"
 
+[tool.ruff.pydocstyle]
+convention = "google"
+
+[tool.ruff.per-file-ignores]
+"new_arrivals_chi/migrations/*" = ["D103"]
```

Comment:
Removes the `D103` rule from applying to the `new_arrivals_chi/migrations/*` folder.